pandas/tests/series/test_missing.py

```diff
@@ -1061,21 +1068,36 @@ def test_interp_limit(self):
         result = s.interpolate(method='linear', limit=2)
         assert_series_equal(result, expected)
 
-        # GH 9217, make sure limit is an int and greater than 0
-        methods = ['linear', 'time', 'index', 'values', 'nearest', 'zero',
-                   'slinear', 'quadratic', 'cubic', 'barycentric', 'krogh',
-                   'polynomial', 'spline', 'piecewise_polynomial', None,
-                   'from_derivatives', 'pchip', 'akima']
+    @pytest.mark.parametrize("method", NONTEMPORAL_INTERPOLATE_METHODS)
```

Comment:
Let's use a fixture for this.

Reply:
I understand where you’re coming from, with the same `”method”` parameterization in the two tests.  I guess the fixture would be called `nontemporal_method_name` in this approach?

Since this test uses two parameters (`”method”` and `”limit”`), I think it’s a bit clearer to simply have two parallel pytest forms (in this case, two usages of `pytest.mark.parmeterize`), to make it clear that the Cartesian product of parameters is being used.

Reply:
From a maintenance perspective, fixture is much preferred over multiple usages of the parameter.

> I guess the fixture would be called `nontemporal_method_name` in this approach?

Yes, that would be reasonable.